setup.py

```diff
@@ -28,21 +28,21 @@ def generate_cython_stub_file(pyx_filepath: str, output_filepath: str) -> None:
     ignore_pattern = re.compile(r"__cinit__\(|__del__\(")
 
     # identify class or function declarations
-    decorator = r"^\s*@.*\n"
-    declaration = r"^\s*(?:class|def)\s+.*(?:.|\n)\n"
-    docstring_double = r"\"\"\"(?:.|\n)*?\"\"\""
-    docstring_single = r"'''(?:.|\n)*?'''"
-    docstring = rf"\s*(?:{docstring_double}|{docstring_single})\s*\n"
-    pattern = re.compile(rf"({decorator})?({declaration})({docstring})?", re.MULTILINE)
+    decorator = r"^\s*@.*?\n"
+    declaration = r"^\s*(?:class|def)\s+.*?:\s*\n"
```

Comment:
The problem comes down to this line: I forgot to match a trailing ":" character in a class/def declaration.